remotespark/utils/constants.py

```diff
@@ -60,15 +60,16 @@
 NOT_STARTED_SESSION_STATUS = "not_started"
 STARTING_SESSION_STATUS = "starting"
 BUSY_SESSION_STATUS = "busy"
+SUCCESS_SESSION_STATUS = "success"
```

Comment:
Talked to him just now and he says a session should not be but might be momentarily in success state. He said we should add it and treat it as a final status.